content/references/protocol-reference/ledger-data/ledger-object-types/accountroot.md

```diff
@@ -57,8 +57,9 @@ The `AccountRoot` object has the following fields:
 | `TicketCount`                 | Number    | UInt32            | No        | How many [Tickets](tickets.html) this account owns in the ledger. This is updated automatically to ensure that the account stays within the hard limit of 250 Tickets at a time. This field is omitted if the account has zero Tickets. _(Added by the [TicketBatch amendment][].)_ |
 | `TickSize`                    | Number    | UInt8             | No        | How many significant digits to use for exchange rates of Offers involving currencies issued by this address. Valid values are `3` to `15`, inclusive. _(Added by the [TickSize amendment][].)_ |
 | `TransferRate`                | Number    | UInt32            | No        | A [transfer fee](transfer-fees.html) to charge other users for sending currency issued by this account to each other. |
-| `WalletLocator`               | String    | Hash256           | No        | **DEPRECATED**. Do not use. |
-| `WalletSize`                  | Number    | UInt32            | No        | **DEPRECATED**. Do not use. |
+| `WalletLocator`               | String    | Hash256           | No        | An arbitrary 256-bit value that users can set. |
+| `WalletSize`                  | Number    | UInt32            | No        | Unused. (The code supports this field but there is no way to set it.) |
```

Comment:
These changes are unrelated to AMM; they match some work from #1517 which updated the transaction but not the object definition. I did them "while I was here" since I was already modifying this page.